org.engplayer.EngPlayer.yml

```diff
@@ -0,0 +1,66 @@
+app-id: org.engplayer.EngPlayer
+runtime: org.gnome.Platform
+runtime-version: '48'
+sdk: org.gnome.Sdk
+command: engplayer
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=dri
+  - --filesystem=home
+  - --filesystem=xdg-run/dconf
+  - --filesystem=~/.config/dconf:ro
+  - --filesystem=/mnt
+  - --filesystem=/run/media
+  - --filesystem=xdg-run/gvfs
+  - --filesystem=xdg-run/gvfsd
+  - --talk-name=org.gtk.vfs.*
+  - --talk-name=ca.desrt.dconf
+  - --env=DCONF_USER_CONFIG_DIR=.config/dconf
+  - --talk-name=org.freedesktop.ScreenSaver
+  - --talk-name=org.freedesktop.Flatpak
+
+modules:
+  - python3-requirements.json
+
+  - name: engplayer
+    buildsystem: simple
+    build-commands:
```

Comment:
all of that is too complicated. you should have a build system upstream to deal with that as it is relevant to anyone building it from source.